src/Tribe/Assets.php

```diff
@@ -99,7 +99,7 @@ public function admin_enqueue_scripts() {
 				[ 'event-tickets-attendees-list-js', 'attendees-list.js', [ 'jquery' ] ],
 				[ 'event-tickets-admin-accordion-js', 'accordion.js', [] ],
 				[ 'event-tickets-admin-accordion-css', 'accordion.css', [] ],
-				[ 'event-tickets-admin-js', 'tickets.js', [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ] ],
+				[ 'event-tickets-admin-js', 'tickets.js', [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ], 'admin_enqueue_scripts', [ 'priority' => 99 ], ],
```

Comment:
* :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 16
* :no_entry_sign: Comma not allowed after last value in single-line array declaration
* :no_entry_sign: Comma not allowed after last value in single-line array declaration

<sub>hash: 5e0641888758862f9b3d0064e97bd97c</sub>